File: Example/iOS-Objc/macOS-Objc/ViewController.m
//
//  ViewController.m
//  macOS-Objc
//
//  Created by nice on 25/10/2019.
//  Copyright © 2019 nice. All rights reserved.
//

#import "ViewController.h"
#import <YouboraConfigUtils/YouboraConfigUtils-Swift.h>

@interface ViewController()

@property YouboraConfigViewController *configViewController;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.configViewController = [YouboraConfigViewController initFromXIB];
    
    [self.configViewController insertIntoContainerWithContainerView:self.container];

    // Do any additional setup after loading the view.
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}


@end

File: Example/iOS/macOS/ViewController.swift
//
//  ViewController.swift
//  macOS
//
//  Created by nice on 14/10/2019.
//  Copyright © 2019 nice. All rights reserved.
//

import Cocoa
import YouboraConfigUtils_macOS

class ViewController: NSViewController {
    
    let child = YouboraConfigViewController().initFromXIB()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        child.insertIntoParent(parentViewController: self)
        
        // Do any additional setup after loading the view.
    }
}
